client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -419,12 +441,27 @@ export const SeoForm = React.createClass( {
 					</Notice>
 				}
 
+				{ showAdvancedSeo && isJetpackUnsupported &&
```

Comment:
Maybe we should add Jetpack version check to `showAdvancedSeo` in order to hide form sections for older versions.